libs/platform/form/combobox/commons/base-combobox.ts

```diff
@@ -165,9 +165,12 @@ export abstract class BaseCombobox
 
     /** Datasource for suggestion list */
     @Input()
-    set dataSource(value: FdpComboBoxDataSource<any>) {
-        if (value) {
+    set dataSource(value: FdpComboBoxDataSource<any> | null) {
+        if (value !== null) {
             this._initializeDataSource(value);
```

Comment:
this is not the same as before. With `if (value !== null)` you are not checking if value is false, 0, "", undefined. If that's the case, ok.